### ISSUE TITLE
Modify root presentation master

### ISSUE THREAD

**Issue by singerla** [singerla/pptx-automizer#7]
As already mentioned in #6, it should also be possible to edit the presentation master(s). As you can have different masters, it needs to have a selector as well. Syntax would be similar:

```js
let pres = automizer.loadRoot(`RootTemplate.pptx`);

// We would also like to modify an existing slide master:
pres.modifySlideMaster(1, (slideMaster) => {
  // "Placeholder 1" has to be the name of an existing element on slideMaster #1 in RootTemplate.pptx
  slideMaster.modifyElement('Placeholder 1', [
    modify.setContent("My custom placeholder")
  ]);
});
```

This has a slightly higher effort, because pptx-automizer cannot handle slide masters at all so far.

**Comment by singerla** [comment]
See also #52

**Comment by singerla** [comment]
This was implemented with #53
